_posts/2023-02-16-communications-netiquette.md

```diff
@@ -0,0 +1,153 @@
+
+If there is an on-going conversation in a topic specific channel and we want to request the attention of a team, we can do it by starting to write **_@_** and searching for the team. That will notify everyone in that team.
+
+If a voice conversation is held about it, sharing the meeting link can be an invitation for others to join the discussion.
+
+## Direct Messages
+
+Every time we write a direct message we should ask to ourselves: is this information really private?
+
+Discussing work topics in private messages is a bad idea for several reasons, the main two being:
+
+**n-plication of information and time**: if a single person has the answer and others ask in private, that person will need to spend the time to answer to the other n. Not only that, if a new person comes to the company, it is most probable that this question will be made once more.
+
+**availability of information**: what we ask in a direct message the information will be only available to the two people in that conversation. It will not appear in searches by another people, and if someone else needs to be included in the conversation they are lacking the context.
+
+It takes some inner gym to get there, but it is a good habit to redirect to a channel a question if we are asked in a private message. It is kind gesture for the whole team.
+
+## Channel life cycles
+
+Following the conventions within this agreements, channels can be created and archived at discretion. Timely gardening of the channels will be required.
+
+# Meetings
+
+## Attention focus
+
+Meetings are moments of collaboration, where everyone&#39;s attention should be fully dedicated to the meeting as long as it lasts. This means that as much as possible, disable slack notifications while you are in a meeting, and update your slack status communicating to the team that you are busy in a meeting.
+
+If a person can foresee that will not be able to pay full attention to the meeting, it is preferable to skip or postpone the meeting instead of not being present.
+
+## Scheduling
+
+Meetings should be scheduled in advance and agreed by all the assistants, specially if it is a one-on-one meeting.
+
+For ad-hoc meeting requests, the same rule apply by asking about availability over slack.  If a person is dedicating their full attention to another task, creating a zoom call before agreeing on starting the meeting creates a tension and a distraction and it must be avoided.
+
+The topic of the meeting should also be clarified in advance, and ideally also the agenda and desired outcomes.
+
+As a principle, we should avoid meetings where people disconnect because they are not involved. So if people feel disconnected or bored in a meeting, something about the scheduling went wrong.
+
+Meetings should be kept under 45 minutes to make it possible for people to stay focused. Scheduling meetings on Thursdays is discouraged because of [Focus Thursdays](https://bahamut.slab.com/posts/focus-thurs-day-qep21nok).
```

Comment:
We will see this particular issue in other post coming from Bahamut, what should we do with the internal links to the wiki to documents that haven't been published? The current unmodified link will take us to the login of Bahamut, maybe we want to add a label or emoji that indicates that it is still not available or add a foot note to briefly explain what the document is about.

Reply:
Yes I am going back and fourth between simply publishing on slab vs integrating here. But I think integrating will be better, like this its all open source on github and in one place. So I will substitute it with a relative link and then we can add the focus thursdays one next.